### PR TITLE
API: Adds more information to the language API

### PR DESCRIPTION
**Pull request by SaptakS**
Refs: #3233 

- [x] Every commit has a message which describes it
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them
- [x] Any code changes come with test case
- [x] Any new functionality is covered by user documentation

### CODE REVIEW

File: weblate/lang/models.py [hunk 1]
```diff
@@ -169,6 +169,9 @@ def aliases_get(self, code, expanded_code=None):
                     return ret
         return None
 
+    def get_aliases_names(self, code):
+        return [alias for alias, codename in ALIASES.items() if codename == code]
```
Comment (nijel) [inline weblate/lang/models.py:173]: This probably more belongs to the `Language` class, using self.code direclty.

File: weblate/api/serializers.py [hunk 2]
```diff
@@ -90,15 +90,35 @@ def __init__(self, *args, **kwargs):
...
+            "web_url",
+            "url",
+            "statistics_url",
+        )
         extra_kwargs = {
             "url": {"view_name": "api:language-detail", "lookup_field": "code"},
             "code": {"validators": []},
         }
 
+    def get_plural(self, obj):
```
Comment (nijel) [inline weblate/api/serializers.py:116]:
```suggestion
    def get_plurals(self, obj):
```

It returns all plural rules, not just one ;-)

File: weblate/api/serializers.py [hunk 3]
```diff
@@ -90,15 +90,35 @@ def __init__(self, *args, **kwargs):
 
 class LanguageSerializer(serializers.ModelSerializer):
     web_url = AbsoluteURLField(source="get_absolute_url", read_only=True)
+    plural = serializers.SerializerMethodField()
```
Comment (nijel) [inline weblate/api/serializers.py:93]: This needs own serializer to support creating and updating plurals (in future, you don't have to implement that in this PR).

Comment (SaptakS) [reply weblate/api/serializers.py:93]: Should I make it into a HyperLinkedIdentityField? Or just call the serializer with many=True?

Comment (nijel) [reply weblate/api/serializers.py:93]: I think serializer with many is good here, these are probably not useful standalone to deserve own API endpoint.

**Comment by SaptakS** [comment]
www.celeryproject.org seems to be unresponsive.

File: weblate/api/serializers.py [hunk 4]
```diff
@@ -88,12 +88,43 @@ def __init__(self, *args, **kwargs):
...
+            "source",
+            "number",
+            "formula",
+            "type",
+        )
+
+
+class LanguageAliasesSerializer(serializers.ListField):
+    def get_attribute(self, instance):
+        return super().get_attribute(instance)
```
Comment (nijel) [inline weblate/api/serializers.py:105]: Is this needed?

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**